packages/react-native-web/src/index.js

```diff
@@ -13,9 +13,7 @@ export { default as Animated } from './exports/Animated';
 export { default as Appearance } from './exports/Appearance';
 export { default as AppRegistry } from './exports/AppRegistry';
 export { default as AppState } from './exports/AppState';
-export { default as BackHandler } from './exports/BackHandler';
```

Comment:
I briefly tested the canary build by changing package.json and yarn patching the babel plugin to match the changes in this PR.

I may be missing another step, but what is the expectation for code that previously expected ReactNativeWeb to match ReactNative's exports?  

I can work around first party code by creating platform-specific files, but libraries such as react-native-screens will fail until updated for the same reason.

Reply:
RNW hasn't matched the RN exports for years. This is just removing the last of the platform-specific exports.

You can open an issue against their project and ask them to fix it. Looking at their code using BackHandler, it might not have crashed before but they have no special handling for web so functionally it probably wasn't cross-platform anyway. They also use it in only 2 places so it should be straightforward to fix.